experiments/functional/deprovision-multiple-volumes/test.yml

```diff
@@ -0,0 +1,121 @@
+---
+- hosts: localhost
+  connection: local
+
+  vars_files:
+    - test_vars.yml
+
+  tasks:
+    - block:
+
+          ## Generating the testname for deployment
+        - include_tasks: /utils/fcm/create_testname.yml
+
+        ## RECORD START-OF-TEST IN LITMUS RESULT CR
+        - include_tasks: "/utils/fcm/update_litmus_result_resource.yml"
+          vars:
+            status: 'SOT'            
+         
+        ## VERIFY AVAILABILITY OF SELECTED STORAGE CLASS
+        - name: Check whether the provider storageclass is provided.
+          shell: kubectl get sc {{ lookup('env','PROVIDER_STORAGE_CLASS') }}
+          args:
+            executable: /bin/bash
+          register: result
+
+        - name: Create namespace for deployment.
+          shell: kubectl create ns {{ namespace }}
+          args:
+            executable: /bin/bash
+          register: app_ns
+          failed_when: "'created' not in app_ns.stdout"
+
+        - name: Replace the storageclass placeholder with provider
+          replace:
+            path: "{{ pvc_deployment }}"
+            regexp: "testclass"
+            replace: "{{ lookup('env','PROVIDER_STORAGE_CLASS') }}"
+
+        ## Setting up iteration value to generate continuos values ##
+        - set_fact:
+            i: 0
+
+        ## Required application and PVC creation tasks ##
+        - include_tasks: "{{ pvc_creation_tasks }}"
+          with_sequence: start=0 count={{ pvc_count }}
+
+        - name: Obtain the cstor pool pod list
+          shell: >
+            kubectl get pods -n {{ operator_ns }} -l openebs.io/storage-pool-claim=cstor-block-disk-pool 
+            -o custom-columns=:.metadata.name --no-headers
+          args:
+            executable: /bin/bash
+          register: pool_pods
+
+        - name: Obtain the PV name
+          shell: >
+             kubectl get pvc -n {{ namespace }} -o custom-columns=:.spec.volumeName --no-headers
+          args:
+            executable: /bin/bash
+          register: pv_name
+
+        - name: Delete the all PVC and pool pods
+          shell: >
+             kubectl delete pods -n {{ operator_ns }} {{ pool_pods.stdout_lines[0] }} && kubectl delete pvc -n {{ namespace }} --all
+          args:
+            executable: /bin/bash
+          register: pod_pvc_delete
+
+        - name: Check if the PVCs are deleted
+          shell: kubectl get pvc -n {{ namespace }}
+          args:
+            executable: /bin/bash
+          register: pvc_status
+          until: "'No resources found.' in pvc_status.stderr"
+          delay: 5
+          retries: 30
+
```

Comment:
we need to check CVRs as well. Fail if CVR deletion doesn't happen due to finalizers